### DISCUSSION TITLE
Table option wont' work

### DISCUSSION THREAD

**Discussion by sundsx** [icflorescu/mantine-datatable#524] [Q&A]
The options table wont work**

Also the table work only with Chrome- Error from FF and Safari

this my table setup
```
  <DataTable
        striped // -> not striped the rows
        withTableBorder // -> not border the table
        withColumnBorders // -> no column border appear
        borderRadius="sm" // -> no border radius
        highlightOnHover // -> no highlight the rows
        verticalSpacing="xs" // -> works well
        fz="xs" // -> works well
        verticalAlign="center" // -> works well

        rowExpansion={} // -> works well
```

Chrome with no options rendered
<img width="1841" alt="Schermata 2024-01-20 alle 11 18 37" src="https://github.com/icflorescu/mantine-datatable/assets/1713528/4fcb3bb9-598a-49db-bd81-6a3142e59633">

Firefox Error
<img width="1889" alt="Schermata 2024-01-20 alle 11 19 23" src="https://github.com/icflorescu/mantine-datatable/assets/1713528/85c47b58-e4d0-403e-86c8-a38d68b93511">

I have Mantine 7 in React application

My index tsx

```
import React from 'react';
import ReactDOM from 'react-dom/client';
import App from './App';
import { createTheme, MantineProvider } from '@mantine/core';
import '@mantine/core/styles.css';
import '@mantine/core/styles.layer.css';
import 'mantine-datatable/styles.layer.css';
import './layout.css';
```

```
const theme = createTheme({
  /** Put your mantine theme override here */
});

const root = ReactDOM.createRoot(
  document.getElementById('root') as HTMLElement
);
root.render(
  <MantineProvider theme={theme} defaultColorScheme="dark">
  <React.StrictMode>
    <App />
  </React.StrictMode>
  </MantineProvider>
);
```

So can help me to sovle this issue

Thanks

**Comment by icflorescu** [comment]
Hey @sundsx,

You can check that `striped`, `withTableBorder` `withColumnBorders`, `borderRadius` and `highlightOnHover` attributes work perfectly on this page in the documentation (which is actually a Next.js application):

https://icflorescu.github.io/mantine-datatable/examples/basic-table-properties/

As you can see, that page works in all browsers - which means there must be an error somewhere else in your code.
Since it's not a bug in Mantine DataTable, I am going to covert this issue to a discussion and perhaps somebody will be able to point you in the right direction.

Related to your code, I don't understand:
- why are you importing the Mantine core styles twice - **once the standard version**, **once the layers version**?
- How are you controlling the order of layers? What's the content of your `layout.css`?

**Comment by icflorescu** [comment]
@sundsx - I've converted this to a discussion in Q&A. Maybe some user will be able to point you in the right direction.

**Comment by icflorescu** [comment]
Make sure to carefully read the styling docs page (https://icflorescu.github.io/mantine-datatable/styling/) and [search other discussions here](https://github.com/icflorescu/mantine-datatable/discussions). I remember somebody mentioning a similar case a while ago, and in the end there was an issue of improper styles configuration.
